Tribler/Core/Modules/restapi/metadata_endpoint.py

```diff
@@ -101,7 +101,7 @@ def render_GET(self, request):
             channels_list = [channel.to_simple_dict() for channel in channels]
 
         return json.dumps({
-            "channels": channels_list,
+            "results": channels_list,
```

Comment:
I am not entirely sure whether I agree with this change. On one hand, it is more consistent. On the other hand, a `results` key is quite generic. What is the motivation behind this change?

Reply:
The reason is the big refactoring of `triblertablecontrollers.py`. Having the API consistent across different endpoints really helps with inheritance and code reuse. Both for API writers and users.